packages/@aws-cdk-testing/framework-integ/test/aws-codebuild/test/integ.project-windows-image.ts

```diff
@@ -25,8 +25,7 @@ new IntegTest(app, 'integ-project-windows-images', {
     new ImageTestStack(app, 'WinCore2019-1', WindowsBuildImage.WIN_SERVER_CORE_2019_BASE),
     new ImageTestStack(app, 'WinCore2019-2', WindowsBuildImage.WIN_SERVER_CORE_2019_BASE_2_0),
     new ImageTestStack(app, 'WinCore2019-3', WindowsBuildImage.WIN_SERVER_CORE_2019_BASE_3_0),
-    // Windows 2022 images are not supported for on-demand build projects
-    /* new ImageTestStack(app, 'WinCore2022-3', WindowsBuildImage.WIN_SERVER_CORE_2022_BASE_3_0), */
+    new ImageTestStack(app, 'WinCore2022-3', WindowsBuildImage.WIN_SERVER_CORE_2022_BASE_3_0),
```

Comment:
Is there an authoritative list for this? I'm thinking about how we maintain this list going forward when new versions are added.

Reply:
There's an API that can be queried. It doesn't contain this split of fleet vs on-demand that this PR is removing. But then again, I'm not sure why that split existed in the first place.

[codebuild:ListCuratedEnvironmentImages](https://docs.aws.amazon.com/codebuild/latest/APIReference/API_ListCuratedEnvironmentImages.html) API command